frontend/cypress/integration/featureFiles/istio_config.feature

```diff
@@ -9,50 +10,261 @@ Feature: Kiali Istio Config page
     And user is at the "istio" page
     And user selects the "bookinfo" namespace
 
-  @istio-page
   Scenario: See all Istio Config objects in the bookinfo namespace.
     Then user sees all the Istio Config objects in the bookinfo namespace
     And user sees Name information for Istio objects
     And user sees Namespace information for Istio objects
     And user sees Type information for Istio objects
     And user sees Configuration information for Istio objects
-  
-  @istio-page
+
   Scenario: Filter Istio Config objects by Istio Name
     When the user filters by "Istio Name" for "bookinfo-gateway"
     Then user only sees "bookinfo-gateway"
 
-  @istio-page
   Scenario: Filter Istio Config objects by Istio Type
     When the user filters by "Istio Type" for "Gateway"
     Then user only sees "bookinfo-gateway"
 
-  @istio-page
   Scenario: Filter Istio Config objects by Valid configuration
     When the user filters by "Config" for "Valid"
     Then user sees "bookinfo-gateway"
     And user sees "bookinfo"
 
-  @istio-page
   Scenario: Ability to create an AuthorizationPolicy object
     Then the user can create a "AuthorizationPolicy" Istio object
 
-  @istio-page
   Scenario: Ability to create a Gateway object
     Then the user can create a "Gateway" Istio object
 
-  @istio-page
   Scenario: Ability to create a PeerAuthentication object
     Then the user can create a "PeerAuthentication" Istio object
 
-  @istio-page
   Scenario: Ability to create a RequestAuthentication object
     Then the user can create a "RequestAuthentication" Istio object
 
-  @istio-page
   Scenario: Ability to create a ServiceEntry object
     Then the user can create a "ServiceEntry" Istio object
 
-  @istio-page
   Scenario: Ability to create a Sidecar object
     Then the user can create a "Sidecar" Istio object
+
+  @crd-validation
+  Scenario: KIA0101 validation
+    Given a "foo" AuthorizationPolicy in the "bookinfo" namespace
+    And the AuthorizationPolicy has a from-source rule for "bar" namespace
+    When the user refreshes the list page
+    And user selects the "bookinfo" namespace
+    Then the AuthorizationPolicy should have a "warning"
+
+  @crd-validation
+  Scenario: KIA0102 validation
+    Given a "foo" AuthorizationPolicy in the "bookinfo" namespace
+    And the AuthorizationPolicy has a to-operation rule with "non-fully-qualified-grpc" method
+    When the user refreshes the list page
+    And user selects the "bookinfo" namespace
+    Then the AuthorizationPolicy should have a "warning"
+
+  @crd-validation
+  Scenario: KIA0104 validation
+    Given a "foo" AuthorizationPolicy in the "bookinfo" namespace
+    And the AuthorizationPolicy has a to-operation rule with "missing.hostname" host
+    When the user refreshes the list page
+    And user selects the "bookinfo" namespace
+    Then the AuthorizationPolicy should have a "danger"
```

Comment:
This test is failing for me locally even though it passes in CI:

![Screenshot from 2022-06-03 12-10-36](https://user-images.githubusercontent.com/6226732/171905143-ca63cf36-835d-4a9c-9fbd-7b1952f65620.png)

I couldn't get a screenshot in time but it looks like the config is all green as opposed to being "danger". Could that just be an issue with my env?

Reply:
You may need to apply the patch that I added in the `install-testing-demos.sh` file:

https://github.com/kiali/kiali/pull/5085/files?show-deleted-files=true&show-viewed-files=true&file-filters%5B%5D=#diff-ca354ddefc935cbbb001f2dc17df036af93fc0a70a24f2dc65e8759281584fad

Reply:
I'll re-run that script and give it another shot.

Reply:
Failure went away after re-running this script! There's some additional istio config elements that stick around after the tests causing other test failures after fixing the passthrough cluster issues but those tests are probably making bad assumptions about there being a specific number of elements in a table.